build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
 		}
 	}
 	dependencies {
-		classpath "net.fabricmc:weave:0.4.0.19"
+		classpath "cuchaz:enigma:0.14.0.122"
```

Comment:
Mind using a `+` to keep up with future updates?

Reply:
shouldn't really imo. That way if an enigma update breaks people can still use yarn as if nothing happened

Reply:
eh fine